config/tests/jobs/README.md

```diff
@@ -8,4 +8,7 @@ To run via bazel: `bazel test //config/tests/jobs/...`
 
 To run via go: `go test .`
 
+If tests fail, re-run with the `UPDATE_FIXTURE_DATA=true` env variable
+and include the modified files in the PR which updates the jobs.
```

Comment:
we should probably have a convenient make target since people will be doing this often, normally we have verify scripts spit out the command people need to run when there's generated data to update

Reply:
I added `make update-config-fixtures` plus some other make targets like `update-unit` for symmetry with `make unit` which happens to run the Go test. The Go test itself now points to `make update-config-fixtures` because that is faster when the goal is to update the file.